pkg/server/helpers_selinux_test.go

```diff
@@ -65,7 +85,12 @@ func TestInitSelinuxOpts(t *testing.T) {
 		t.Logf("TestCase %q", desc)
 		processLabel, mountLabel, err := initSelinuxOpts(test.selinuxOpt)
 		assert.NoError(t, err)
-		assert.Equal(t, test.processLabel, processLabel)
-		assert.Contains(t, test.mountLabels, mountLabel)
+		if desc == "testUser" {
+			assert.Equal(t, test.processLabel, processLabel)
+			assert.Contains(t, test.mountLabels, mountLabel)
+		} else {
+			assert.Regexp(t, "^"+test.processLabel, processLabel)
+			assert.Regexp(t, "^"+test.mountLabels[0], mountLabel)
```

Comment:
What about `test.mountLabels[1]`?

Reply:
Only `test.mountLabels[0]` is valid values except last one.

Reply:
Then why do we have it in the test case? Seems that we don't use it at all.

Reply:
It will be used in `assert.Contains(t, test.mountLabels, mountLabel)`